<ui version="4.0" >
 <class>SelectAvatarDialogClass</class>
 <widget class="QDialog" name="SelectAvatarDialogClass" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>412</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="sizeConstraint" >
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="frameVLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>5</number>
        </property>
        <property name="bottomMargin" >
         <number>16</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeToolButton" >
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="accessibleName" >
           <string>transparentButton</string>
          </property>
          <property name="layoutDirection" >
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="popupMode" >
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="mainDialogFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <property name="leftMargin" >
          <number>12</number>
         </property>
         <property name="rightMargin" >
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <property name="topMargin" >
            <number>2</number>
           </property>
           <property name="bottomMargin" >
            <number>11</number>
           </property>
           <item>
            <widget class="QLabel" name="topMessageLabel" >
             <property name="text" >
              <string>TopLabel</string>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addAvatarButton" >
             <property name="minimumSize" >
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="accessibleName" >
              <string>rectButton</string>
             </property>
             <property name="text" >
              <string>AddAvatarButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="avatarListWidget" >
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth" >
            <number>0</number>
           </property>
           <property name="showDropIndicator" stdset="0" >
            <bool>false</bool>
           </property>
           <property name="dragDropMode" >
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="iconSize" >
            <size>
             <width>72</width>
             <height>72</height>
            </size>
           </property>
           <property name="verticalScrollMode" >
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="movement" >
            <enum>QListView::Static</enum>
           </property>
           <property name="flow" >
            <enum>QListView::LeftToRight</enum>
           </property>
           <property name="isWrapping" stdset="0" >
            <bool>true</bool>
           </property>
           <property name="resizeMode" >
            <enum>QListView::Adjust</enum>
           </property>
           <property name="layoutMode" >
            <enum>QListView::SinglePass</enum>
           </property>
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="viewMode" >
            <enum>QListView::IconMode</enum>
           </property>
           <property name="uniformItemSizes" >
            <bool>true</bool>
           </property>
           <property name="selectionRectVisible" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameBottom" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okButton" >
           <property name="accessibleName" >
            <string>rectButton</string>
           </property>
           <property name="text" >
            <string>OkButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton" >
           <property name="accessibleName" >
            <string>rectButton</string>
           </property>
           <property name="text" >
            <string>CancelButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources/>
 <connections/>
</ui>
